checkers/typescript/avoid_using_any_type.yml

```diff
@@ -0,0 +1,16 @@
+language: ts
+name: avoid_using_any_type
+message: Detected the usage of type "any" better to use "unknown" type
+category: security
+
+pattern: |
+  (variable_declarator
+    type: (_
+        (predefined_type) @any_type (#eq @any_type "any")
+      )
+    
+  ) @avoid_using_any_type
+
+description: |
+  In TypeScript, "any" allows a variable to hold any value without type checking, essentially disabling type safety, while "unknown" represents a value of unknown type but forces you to explicitly check and narrow its type before using it, promoting type safety; essentially, "any" is very flexible but less safe, while "unknown" is more restrictive but safer when dealing with uncertain data types
```

Comment:
Add examples of good and bad patterns. Use backticks instead of quotes to denote `any` and `unknown` and any other code.